data/hai/hai missions.txt

```diff
@@ -1808,3 +1846,399 @@ mission "Nanachi Meeting"
+				goto next
+			
+			label coalition
+			`	"I hope the Coalition treated you well," one of the elders says.`
+			`	"They were an interesting group. Very enthusiatic about meeting new people, although very touchy on the subject of the Quarg."`
+			
+			label next
+			`	Yashili then hands the copy of the data to Osen. "You'll be needing this," she says.`
+			`	"And I'll certainly love it," Osen responds. "The spaceport workers have already sent the cargo en route to my shipyard. I'll begin working on implementing it into the Pond Strider immediately."`
+			choice
+				`	"Pond Strider?"`
+				`	"I hope you have fun with that."`
+					goto fun
+			
+			`	"Yes. That is the name that we settled on for the ship, and its drones will be named the Flea."`
+			`	"I'm getting itchy just thinking about it," one of the elders jokes.`
+				goto end
+			
+			label fun
+			`	"It's hard not to have fun when you love doing your job."`
+			
+			label end
+			`	Osen is about to run off, but one of the elders stops him. "Can you give us any estimate on how long it will take to make use of this data?"`
+			`	"Oh, yes," Osen says. "It may take us a few months to get everything working. A single month at the least if we are lucky."`
+			`	"This is good to know. The sooner we have these ships function, the better."`
+			`	Osen once again gets ready to run off, but this time stops himself and turns to you. "Thank you Captain. Without this information, it could have taken me years to finish this project." You nod in response, and Osen finally runs off to his shipyard.`
+			`	"I would like to thank you as well," Yashili says. "It has been some time since I have had the chance to leave our space. It is always exciting to go to new places and meet new people." Yashili bows to you, and you return a bow of your own.`
+			`	"And we also need to express our gratitude," one of the elders says. "It is good to have you as a friend of the Hai. We shall keep you no longer, though. May you stay safe on your adventures."`
+				decline
+
+
+
+mission "Strider notification"
+	invisible
+	landing
+	source
+		government "Hai"
+	to offer
+		has "event: pond strider mass production"
+	on offer
+		conversation
+			scene `thumbnail/hai pond strider`
+			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
```

Comment:
The second last sentence should read:
I won't **spoil** the surprise for you though.